tests/test_data/GCA_000692775_1_trunc2.faa

```diff
@@ -0,0 +1,2469 @@
+>KK736506.1_1 # 3 # 746 # 1 # ID=1_1;partial=10;start_type=Edge;rbs_motif=None;rbs_spacer=None;gc_cont=0.512
```

Comment:
I'm wondering whether we should provide gzipped files

Reply:
can do